src/components/Description.jsx

```diff
@@ -0,0 +1,29 @@
+import './scss/Description.scss';
```

Comment:
This should not appear on this PR, 'cause it was already on the previous one è_é